### PR TITLE
add field description in types.go

### PR DESCRIPTION
**Pull request by allencloud**
add field description in types.go
Signed-off-by: allencloud <allen.sun@daocloud.io>

### CODE REVIEW

File: types/types.go [hunk 1]
```diff
@@ -367,14 +367,14 @@ type NetworkSettingsBase struct {
...
-	Gateway             string
-	GlobalIPv6Address   string
-	GlobalIPv6PrefixLen int
-	IPAddress           string
-	IPPrefixLen         int
-	IPv6Gateway         string
-	MacAddress          string
+	EndpointID          string // EndpointID uniquely represents a service endpoint in a Sandbox
+	Gateway             string // Gateway holds the gateway address for the network
+	GlobalIPv6Address   string // GlobalIPv6Address holds netork's global IPv6 address
```
Comment (thaJeztah) [inline types/types.go]: s/netork's/network's/

File: types/types.go [hunk 2]
```diff
@@ -416,16 +416,16 @@ type VolumeCreateRequest struct {
...
-	Scope      string
-	Driver     string
-	EnableIPv6 bool
-	IPAM       network.IPAM
-	Internal   bool
-	Containers map[string]EndpointResource
-	Options    map[string]string
-	Labels     map[string]string
+	Name       string                      // Name is the requested name of the volume
+	ID         string                      `json:"Id"` // ID uniquely indetifies a network on a single machine
```
Comment (thaJeztah) [inline types/types.go]: s/idetifies/identifies/

**Comment by thaJeztah** [comment]
nice! spotted some typo's, but LGTM otherwise

**Comment by allencloud** [comment]
@thaJeztah  
I have updated the PR, and fixed the typos.

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by calavera** [comment]
LGTM

**Pull request merged**